tests/test_network_models.py

```
@@ -0,0 +1,212 @@
+            }
+        )
+
+
+def test_network_configuration() -> None:
+    """Test network configuration validation."""
+    # Test valid configuration
+    config = NetworkConfiguration(**VALID_NETWORK_CONFIG)
+    assert config.name == "Main Network"
+    assert config.purpose == "corporate"
+    assert config.subnet == "192.168.1.0"
+    assert config.vlans is not None and isinstance(config.vlans[0], VLANConfiguration)
```

Ruff (0.8.2)

Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)